runtests.sh

```diff
@@ -238,6 +239,7 @@ do
             doFlake8Format=true
             doPytypeFormat=true
             doMypyFormat=true
+            doCopyRight=true
         ;;
         --disttests)
```

Comment:
Could you please help add a command arg for it? Maybe `--copyright`.
Others look good to me.

Thanks.

Reply:
thanks, new option added.